### ISSUE TITLE
Too much comments in the code.

### ISSUE THREAD

**Issue by NazarbekAld** [mfnalex/shitlib#8]
We should remove every comment in the repo.

**Comment by Reecepbcups** [comment]
i agree

**Comment by OkKloon** [comment]
Yes, I do agree with this! The presence of comments makes everything more cluttered than it needs. Furthermore, I believe that the code documents itself! On top of that, according to a study made by the Internal Revenue Service, code comments waste 4.31690176% of our valuable times, which could be put into writing 50% more shitty code! Thanks for attending my TED talk 😊.
